lale/lib/lale/functions.py

```diff
@@ -21,6 +21,7 @@
 import pandas as pd
 
 try:
+    import pyspark.sql.functions as pysql
```

Comment:
So we don't need their GroupedData variants here?

Reply:
No we would not need them as the base variants working fine on grouped dataframes.